src/main/java/org/spongepowered/api/item/inventory/ContainerTypes.java

```diff
@@ -35,28 +35,35 @@
 
     // Containers backed by an inventory.
 
+    // TODO add new inventories
+
     // SORTFIELDS:ON
 
+    /**
+     * Size 5.
+     */
+    public static final ContainerType BREWING_STAND = DummyObjectProvider.createFor(ContainerType.class, "brewing_stand"); // "minecraft:brewing_stand"
+
     /**
      * Size 27 (3x9 grid)
      */
     public static final ContainerType CHEST_3X9 = DummyObjectProvider.createFor(ContainerType.class, "chest_3x9"); // "minecraft:chest"
```

Comment:
Becomes `GENERIC_3X9`, `GENERIC_6X9` for the one below

Reply:
Where do you get the names from?

Reply:
It's the "menu" registry: https://gist.github.com/Cybermaxke/c91032ba8571bddfaf3048250cbe6d10

Reply:
in code `net.minecraft.inventory.container.ContainerType`